### PR TITLE
Add unit tests for Minimum Path Sum algorithm

### PR DESCRIPTION
**Pull request by ngominhtrint**
Add unit tests for Minimum Path Sum algorithms

Screenshot 2023-07-09 at 12 02 58 PM

- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Java/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized it.
- [x] All filenames are in PascalCase.
- [x] All functions and variable names follow Java naming conventions.
- [ ] All new algorithms have a URL in their comments that points to Wikipedia or other similar explanations.

### CODE REVIEW

File: src/test/java/com/thealgorithms/dynamicprogramming/MinimumPathSumTest.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
+package com.thealgorithms.dynamicprogramming;
+
+import static org.junit.jupiter.api.Assertions.*;
+import org.junit.jupiter.api.Test;
```
Comment (debasishbsws) [inline src/test/java/com/thealgorithms/dynamicprogramming/MinimumPathSumTest.java:4]:
```suggestion
import static org.junit.jupiter.api.Assertions.*;

import org.junit.jupiter.api.Test;
```

Comment (debasishbsws) [reply src/test/java/com/thealgorithms/dynamicprogramming/MinimumPathSumTest.java:4]: To resolve the clang lint

**Review by debasishbsws — CHANGES_REQUESTED** [review]

File: src/test/java/com/thealgorithms/dynamicprogramming/MinimumPathSumTest.java (hunk 1 above)
Comment (ngominhtrint) [reply src/test/java/com/thealgorithms/dynamicprogramming/MinimumPathSumTest.java]: Thanks bro. I've updated.

**Review by debasishbsws — APPROVED** [review]
Looks good thanks.
